terraform-apply/download_plan_file.sh

```diff
@@ -9,14 +9,10 @@ workflow=$PLAN_WORKFLOW_NAME
 
 pr_head_sha=$(jq -r ".head.sha" "$CI_INFO_TEMP_DIR/pr.json")
 
-body=$(gh run list -w "$workflow" -b "$branch" -L 1 --json headSha,databaseId --jq '.[0]')
-run_id=$(echo "$body" | jq -r ".databaseId")
-head_sha=$(echo "$body" | jq -r ".headSha")
+run_id=$(gh run list -c "$pr_head_sha" -w "$workflow" -b "$branch" -L 1 --json databaseId --jq '.[0].databaseId')
 
-if [ "$head_sha" != "$pr_head_sha" ]; then
```

Comment:
We specify a commit hash when downloading a plan file, so we don't have to check a commit hash anymore.